src/uni_modules/wot-ui/components/wd-button/index.scss

```
@@ -18,6 +18,10 @@ $button-primary-plain-bg-active: var(--wot-button-primary-plain-bg-active, $prim
 $button-primary-plain-border: var(--wot-button-primary-plain-border, $primary-6) !default;
 // 主类型 plain 边框激活色
 $button-primary-plain-border-active: var(--wot-button-primary-plain-border-active, $primary-7) !default;
+// 主类型 soft 背景色
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)